<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="slider_red">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_green">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>70</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="slider_blue">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>110</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>256</width>
      <height>256</height>
     </rect>
    </property>
    <property name="text">
     <string>IMAGE</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_threshold">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>150</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_red">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>60</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>RED</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_green">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>60</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>GREEN</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_blue">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>110</y>
      <width>60</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>BLUE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_threshold">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>150</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>THRESHOLD</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_erosion">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>210</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_erosion">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>210</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>EROSION</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_dilatacion">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>180</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_dilatacion">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>180</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>DILATACION</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_erosion2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>270</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_erosion2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>270</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>EROSION</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_dilatacion2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>240</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>DILATACION</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_dilatacion2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>240</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
